Mathlib/Order/WithBot.lean

```diff
@@ -225,6 +225,11 @@ theorem not_coe_le_bot (a : α) : ¬(a : WithBot α) ≤ ⊥ := fun h =>
   Option.not_mem_none _ hb
 #align with_bot.not_coe_le_bot WithBot.not_coe_le_bot
 
+@[simp]
+theorem le_bot_iff : ∀ {a : WithBot α}, a ≤ ⊥ ↔ a = ⊥
+  | (a : α) => by simp [not_coe_le_bot _]
+  | ⊥ => by simp
```

Comment:
Isn't this the same as https://leanprover-community.github.io/mathlib4_docs/Mathlib/Order/BoundedOrder.html#le_bot_iff?

Reply:
This does not need a `PartialOrder`.

Reply:
Can you add a docstring that explains that?